rest/stats_context.go

```diff
@@ -227,4 +242,96 @@ func recordGoroutineHighwaterMark(stats *expvar.Map, numGoroutines uint64) (maxG
 
 	return maxGoroutinesSeen
 
-}
\ No newline at end of file
+}
+
+func networkInterfaceStatsForHostnamePort(hostPort string) (iocountersStats gopsutilnet.IOCountersStat, err error) {
+
+	host, _, err := net.SplitHostPort(hostPort)
+	if err != nil {
+		return iocountersStats, err
+	}
+
+	allInterfaces := false
+	if host == "" || host == "0.0.0.0" {
+		allInterfaces = true
+	}
+
+	// "localhost" -> "127.0.0.1", since this code only works with IP addresses
+	if host == "localhost" {
+		host = "127.0.0.1"
```

Comment:
Not that I know of, but if you know of any specific reason why this might fail, let me know.  As a disclaimer, I've never used ipv6.  Not knowingly at least.

@sridevi-15 does QE do any Sync Gateway testing on ipv6?  Or do you have any suggestions on how I can test with ipv6 locally?

Reply:
I've done a bit of IPv6 testing before with SG. Easiest way is to run both SG and CBS inside Docker. There's a QE guide linked in this comment: https://github.com/couchbase/sync_gateway/pull/3250